packages/synapse-react-client/src/components/EntityAclEditor/EntityAclEditor.tsx

```diff
@@ -318,7 +334,38 @@ const EntityAclEditor = React.forwardRef(function EntityAclEditor(
         isProject={isProject}
         isInherited={updatedIsInherited}
         benefactorId={updatedIsInherited ? parentAcl?.id : entityId}
+        isAfterUpload={isAfterUpload}
       />
+      {showInheritedAclUpdateWarning && (
+        <FullWidthAlert
+          isGlobal={false}
+          variant={'warning'}
+          title={`Edits will affect settings of entire ${entityTypeToFriendlyName(
+            entityBundle?.entityType,
+          ).toLowerCase()}.`}
+          description={
+            <>
+              <p>
+                Editing the settings here will impact the sharing settings for
+                all files and folders within this project, not just the ones
```

Comment:
Should "within this project" be "within this ${entityTypeToFriendlyName(entityBundle?.entityType).toLowerCase()"

Reply:
yep, thanks for catching that!